sw/host/opentitanlib/src/bootstrap/mod.rs

```diff
@@ -68,6 +72,10 @@ impl Bootstrap {
             BootstrapProtocol::Eeprom => {
                 unimplemented!();
             }
+            BootstrapProtocol::Emulator => {
+                // Not intended to be implemented by this struct.
+                unimplemented!();
+            }
```

Comment:
What would it take to implement the emulator protocol in `struct Bootstrap`?  The main workhorse for the bootstrap protocol is expected to be implemented in the `UpdateProtocol` trait.

(my guess is that you're concerned that too much backend-specific stuff will sneak into `Bootstrap` even with the trait trying to keep implementation details separated).